app/models/person.rb

```diff
@@ -61,6 +61,15 @@ class Person < ActiveRecord::Base
   STATUS = ['Leave of Absence', 'Inactive', 'Active', 'Applicant','Prospect','Declined']
   DEPARTMENT = ['Police', 'CERT', 'Other']
 
+  after_initialize do
```

Comment:
Rather than changing the model, what if we changed the controller instead ? 
Adding 
    if Department.active.managing_people.count == 1
      @person.department = Department.active.managing_people.first 
    end
to here =>
https://github.com/ReadyResponder/ReadyResponder/blob/development/app/controllers/people_controller.rb#L87

Reply:
Totally agree with moving this to the controller but why not choosing the department for the model on `new` action when there is only one active department so that the user can see the selected department in the form. If he/she receive the validation error if select the empty department.

Reply:
@kgf I've moved this code into the controller `new` action (during page loading). Please let me know if you want it to be `create` action (during insertion).